cmake/03_compiler_config.cmake

```diff
@@ -171,6 +171,12 @@ if(ENABLE_LINTER)
   endif()
 endif()
 
+# This will deactivate the use of exceptions and use CHECK instead. The whole code should be considered noexcept.
```

Comment:
```suggestion
# This will deactivate the use of exception, and the utility functions we use to wrap throwing exceptions will instead terminate displaying an error message. The whole code should be considered `noexcept`.
```